Source/WebCore/platform/graphics/SourceBufferPrivate.cpp

```diff
@@ -155,6 +155,11 @@ void SourceBufferPrivate::appendCompleted(bool parsingSucceeded, bool isEnded)
 {
     DEBUG_LOG(LOGIDENTIFIER);
 
+    // 5. If the media segment contains data beyond the current duration, then run the duration change algorithm with new
```

Comment:
Maybe add the link to the spec algorithm so we know which algorithms we talk about.

Reply:
done